src/routes/safe/store/actions/transactions/fetchTransactions/loadAllTransactions.ts

```diff
@@ -0,0 +1,81 @@
+import { List, Map } from 'immutable'
+import axios, { AxiosResponse } from 'axios'
+
+import { getAllTxServiceUriTo, getTxServiceHost } from 'src/config'
+import { checksumAddress } from 'src/utils/checksumAddress'
+import { Transaction } from '../../../models/types/transactions'
+
+type ServiceUriParams = {
+  safeAddress: string
+  limit?: number
+  offset?: number
+  orderBy?: string // todo: maybe this should be key of MultiSigTransaction | keyof EthereumTransaction
```

Comment:
not sure to understand the comment

Reply:
because the `orderBy ` parameter is used to order the return fields based on the name of one transaction field (as far I understood the API docs). So basically each one of the keys of `MultiSigTransaction ` / `EthereumTransaction ` could be used

Reply:
it's not 100% guaranteed that api will support sorting by all fields, so this has to be synced with api

Reply:
I think we should wait for implementing once we got the final version of the middleware, what do you think? cc @nicosampler